packages/simplebar-vue/index.vue

```diff
@@ -1,28 +1,30 @@
 <template>
-  <div
-    ref="element"
-  >
+  <div ref="element">
     <div class="simplebar-wrapper">
-        <div class="simplebar-height-auto-observer-wrapper">
-          <div class="simplebar-height-auto-observer" />
-        </div>
-        <div class="simplebar-mask">
-          <div class="simplebar-offset">
-            <div class="simplebar-content-wrapper" ref="scrollElement">
-              <div class="simplebar-content" ref="contentElement">
-                <slot></slot>
-              </div>
+      <div class="simplebar-height-auto-observer-wrapper">
+        <div class="simplebar-height-auto-observer" />
+      </div>
+      <div class="simplebar-mask">
+        <div class="simplebar-offset">
+          <div
+            class="simplebar-content-wrapper"
+            ref="scrollElement"
+            @scroll="$listeners.scroll"
```

Comment:
We need to listen only for `scroll` here, but not for all events as described [here](https://github.com/Grsmto/simplebar/issues/463#issuecomment-638737330), because in case the user will add a click event handler, it should be listened to on the root element, but not on this one. Same about all other events.

I think in case someone will need to listen for other events, he might add his solution with another PR.